desktop/core/src/desktop/conf.py

```diff
@@ -710,7 +710,7 @@ def get_deprecated_login_lock_out_by_combination_browser_user_agent():
     IS_ENABLED=Config(
       key='is_enabled',
       help=_('Turns on Slack application API endpoints'),
-      default=False,
+      default=True,
```

Comment:
Should be false, set it to True in your pseudo-distributed.ini which is local to you

Reply:
Fixed and will keep it in mind